docs/Connecting_to_Settle.md

```diff
@@ -0,0 +1,86 @@
+In this scenario, the `Client Service` is your service and the `API Server`
```

Comment:
Nit: Most of these `code` spans, the ones introducing English technical terms rather than quoting software code/configuration/etc., should be _italics_ instead, to follow typical documentation conventions.